bin/start-ccd-web.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+COMPOSE_FILES="-f docker-compose.yml"
+docker-compose ${COMPOSE_FILES} up ${@} -d ccd-case-management-web \
+                                           ccd-api-gateway \
+                                           idam-api \
+                                           authentication-web \
+                                           smtp-server
```

Comment:
is smtp server needed here?

Reply:
Reason I added it here is because to spin up docker container for smtp server which is then used by idam-api